src/apps/statements/service/storeStatements/queriables/getMetadataFromStatement/getNumericQuestionMetadata.ts

```diff
@@ -0,0 +1,33 @@
+import { get, has } from 'lodash';
+
+import ActivityInteractionType from '../../../../models/ActivityInteractionType';
+import Statement from '../../../../models/Statement';
+
+export const getNumericQuestionMetadata = (statement: Statement)
+  : {readonly [key: string]: any} => {
+  if (
+    !(
+      get(
+        statement.object,
+        ['definition', 'interactionType'],
+      ) === ActivityInteractionType.NUMERIC
+      && has(statement, ['result', 'response'])
+    )
+  ) {
+    return {};
+  }
+
+  const numericQuestionString = get(statement, ['result', 'response']);
+
+  if (!numericQuestionString.includes('[:]')) {
+    return {'https://learninglocker.net/numeric-response': parseFloat(numericQuestionString)};
```

Comment:
Hey @PrinceWaune, Ryan here, we need to replace the dot here with "&46;" because Mongo does not allow dots in keys so we also swap them out for the HTML code.

```suggestion
    return {'https://learninglocker&46;net/numeric-response': parseFloat(numericQuestionString)};
```

Reply:
Hey @ryansmith94. Sure, no problem!